### PR TITLE
 Bug 1162028 - [Messages][New Gaia Architecture] extract conversation/index.html. Moooonkey patching. r=julien, schung

### PR DESCRIPTION
**Pull request by azasypkin**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1063bbc5037ee796576625187eef774159139c28

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4ef40a84134b379494ba734c95e57b211dd25f4c

File: apps/sms/desktop-mock/navigator_moz_contacts.js [hunk 1]
```diff
@@ -156,8 +156,8 @@
   var cid = 0;
   var methods = {
     contains: 'startsWith',
-    match: 'contains',
-    equals: 'contains'
+    match: 'includes',
```
Comment (azasypkin) [inline apps/sms/desktop-mock/navigator_moz_contacts.js]: Leftover from https://bugzilla.mozilla.org/show_bug.cgi?id=1166863

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0011ba8ae4fcbe3fa45c6b94f755b900a39d54d3

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=05265358c53af037fbced062b32f286ec2b72eeb

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=fa957903f9d33e540339bc1a960cf58a8325f03e

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=778cc013fe0a9964034ece70535ba9bc54129ecd

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5bf8e5ada8c51342f63d8bae9bee26f3705803f5

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b50a04c72e9b23da370ccd16c6af5cbc1ab2c633

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3df1bbbd8928a4feecd5f005e86cb643677e339f

**Pull request closed without merging**